.gitlab-ci.yml

```diff
@@ -42,6 +42,17 @@ Deploy app (prod):
     name: prod
     url: https://www.${KUBE_INGRESS_BASE_DOMAIN}
 
+Deploy incubateur redirect pod (prod):
+  extends: .base_docker_kubectl_image_stage
+  variables:
+    K8S_NAMESPACE: www
+    PRODUCTION: "true"
+    script:
+      - echo "kubectl get namespace ${K8S_NAMESPACE}"
+      - kubectl apply -f .k8s/redirect.yml -n ${K8S_NAMESPACE}
+  environment:
+    name: prod
```

Comment:
```suggestion
  extends: 
    - .autodevops_deploy_app_prod
  variables:
    K8S_NAMESPACE: www
    PRODUCTION: "true"
  script:
    - echo "kubectl get namespace ${K8S_NAMESPACE}"
    - kubectl apply -f .k8s/redirect.yml -n ${K8S_NAMESPACE}
```

Reply:
- you can extend the existing [.autodevops_deploy_app_prod](https://github.com/SocialGouv/gitlab-ci-yml/blob/master/autodevops_simple_app.yml#L167-L179)
- correct the indentation of `script`